boards/lilygo_twatch_2020/twatch_landscape.py

```diff
@@ -0,0 +1,13 @@
+from ili9XXX import st7789, REVERSE_LANDSCAPE
+import axp202c
+
+# init power manager, set backlight
+axp = axp202c.PMU()
+axp.enablePower(axp202c.AXP202_LDO2)
+axp.setLDO2Voltage(2800)
+
+# init display
+disp = st7789(
+    mosi=19, clk=18, cs=5, dc=27, rst=-1, backlight=12, power=-1,
+    width=240, height=240, start_x=0, start_y=80,
+    rot=REVERSE_LANDSCAPE, factor=4, initialize=False)
```

Comment:
What if the user wants to initialize twatch but use the async event loop, for example? Or change some other default parameter?
All is that is needed is to pass `asynchronous=True` to the display driver, but since you create the display when the module is imported the user doesn't get a chance.  

Instead, you could create a new `twatch` class with all the default parameters on the constructor and let the user instantiate it. I think that would give much more flexibility.  
The user could write something like:

```python
from twach_board import twach_driver
twatch = twatch_driver(asynchronous=True)
```

What do you think?